client/src/lib/slate-pointers/index.tsx

```diff
@@ -11,7 +11,12 @@ function SlatePointers(options: any = {}) {
       }
 
       const selection = window.getSelection();
-      const range = selection.getRangeAt(0);
+      let range;
+      try {
+        range = selection.getRangeAt(0);
+      } catch (e) {
```

Comment:
Do we need to catch all exceptions here, or can we just catch a specific type and reraise otherwise?

Reply:
Good question. This error was occurring too infrequently and unpredictably for me to figure out what was going on. Also, a lot of code has changed since that happened. I think we should probably just stop swallowing errors here and see if they still occur. If they do hopefully we can figure out why, but there's a chance that the newest approach avoids them entirely.

Reply:
https://github.com/oughtinc/mosaic/pull/145/commits/d22a45803ee9687ae1ebb5e019247f7cf249e9e4